src/tests/test_filesystem.py

```diff
@@ -65,9 +63,7 @@ def test_idbfs_persist_code(selenium_standalone):
     selenium.run_js(
         f"""
         pyodide.runPython(`
-            from importlib import invalidate_caches
             import sys
-            invalidate_caches()
```

Comment:
Hmm test fails without `invalidate_caches`. I suspect this is due to a bug in the IDBFS...